src/executables/util_cs.c

```diff
@@ -3275,51 +3281,59 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
 	      goto check_applied_info_end;
 	    }
-	  error = db_restart (arg->command_name, TRUE, local_database_name);
-	  if (error != NO_ERROR)
+	  if ((error = db_restart (arg->command_name, TRUE, local_database_name)) != NO_ERROR)
 	    {
 	      goto check_applied_info_end;
 	    }
 
 	  if (HA_DISABLED ())
 	    {
+	      check_applied_info = false;
+	      printf ("\nERROR : ");
 	      PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_APPLYINFO,
 						     APPLYINFO_MSG_NOT_HA_MODE));
 	      goto check_applied_info_end;
 	    }
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
-	  (void) db_shutdown ();
-	}
-      else if (check_copied_info)
-	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
+	  error =
+	    la_applyinfo_applied_log_info (database_name, log_path, check_replica_info, verbose, &applied_final_lsa);
+	  if (error != NO_ERROR)
+	    {
+	      check_applied_info = false;
+	      error = NO_ERROR;
+	    }
 	}
 
     check_applied_info_end:
+
       if (error != NO_ERROR)
 	{
-	  PRINT_AND_LOG_ERR_MSG ("%s\n", db_error_string (3));
+	  PRINT_AND_LOG_ERR_MSG ("ERROR : %s\n", db_error_string (3));
+	  check_applied_info = false;
+	  error = NO_ERROR;
+	}
```

Comment:
I think it should be separated. 

It can be confusing what the scope of `check_applied_info_end` is.

Reply:
do you stand for the space(\n)?

Reply:
Oh.. Sorry I forgot this review.
yes, please add a space (\n).

Reply:
haha No problem. Ok thank you.